src/IROperator.h

```diff
@@ -1257,6 +1257,10 @@ inline Expr erf(Expr x) {
     return Internal::halide_erf(std::move(x));
 }
 
+// Fast approximation to some trigonometric functions for Float(32)
```

Comment:
Please write comments using doxygen syntax. Also comment on vectorization and accuracy. See the comment on erf for an example.